.github/workflows/build-and-publish.yml

```diff
@@ -25,7 +25,7 @@ jobs:
             ${{ runner.os }}-node-
 
       - name: setup-docker
-        uses: docker-practice/actions-setup-docker@1.0.9
+        uses: docker-practice/actions-setup-docker@master
```

Comment:
This action [was failing](https://github.com/Brightspace/dynogels/actions/runs/9584203624/job/26427261178) with:

```
install docker
  /usr/bin/sudo apt-get -y install docker-ce=5:20.10.24~3-0~ubuntu-jammy docker-ce-cli=5:20.10.24~3-0~ubuntu-jammy
  Reading package lists...
  Building dependency tree...
  Reading state information...
  Suggested packages:
    aufs-tools cgroupfs-mount | cgroup-lite
  Recommended packages:
    docker-ce-rootless-extras
  The following packages will be DOWNGRADED:
    docker-ce docker-ce-cli
  0 upgraded, 0 newly installed, 2 downgraded, 0 to remove and 61 not upgraded.
  E: Packages were downgraded and -y was used without --allow-downgrades.
  Error: Error: The process '/usr/bin/sudo' failed with exit code 100
```

> Note: I tried first to update to the latest release (1.0.12), but that did not fix the issue